app/backend/controllers/login-signup.js

```diff
@@ -63,16 +55,26 @@ module.exports.signup = async (request, response) => {
   
   // Saves the instance into the database, returns any error occured
   user.save()
-    .then(doc => {
+    .then(doc => {  // when successfully saved the user
     // Omit sensitive data
     const { _id, name, surname, email, location, isTrader, iban, tckn, isPublic, isVerified, googleId } = doc  // Extract certain keys from doc
     
     if(!user.googleId)
-      sendVerifyEmail(email, user.token)
+      // sendVerifyEmail(email, user.token)
```

Comment:
we need to comment it out

Reply:
I've reverted the change, as it was accidental.